torchtext/data/field.py

```diff
@@ -180,11 +213,23 @@ def numericalize(self, arr, device=None, train=True):
 
             if self.postprocessing is not None:
                 arr = self.postprocessing(arr, self.vocab, train)
-        elif self.postprocessing is not None:
-            arr = self.postprocessing(arr, train)
+        else:
+            if self.tensor_type not in self.tensor_types:
+                raise ValueError(
+                    "Specified Field tensor_type {} can not be used with "
+                    "use_vocab=False because we do not know how to numericalize it. "
+                    "Please raise an issue at "
+                    "https://github.com/pytorch/text/issues".format(self.tensor_type))
+            numericalization_func = self.tensor_types[self.tensor_type]
+            # It doesn't make sense to explictly coerce to a numeric type if
```

Comment:
Not too sure how i feel about this part --- what's a use case for passing in sequential data that shouldn't use a vocabulary, and is this (don't do any explicit numericalization, but postprocess it) the proper way to handle it?